cafes/views.py

```diff
@@ -66,6 +67,10 @@ def delete(self, request, cafe_id):
 
         review.delete()
         return JsonResponse({'MESSAGE' : 'REVIEW_DELETED'}, status=204)
+
+class RatingCountView(View):
+    def get(self, request):
+        return JsonResponse({'RATINGS_COUNT' : StarRating.objects.count()}, status=200)
```

Comment:
다음에는 이런 간단한 엔드포인트는 함수형으로 작성해보시면 좋을 것 같습니다~

```suggestion
urlpatterns = [
    path("", count_rating),
    ..
]

def count_rating(self, request):
    return JsonResponse({'RATINGS_COUNT' : StarRating.objects.count()}, status=200)
```